src/test/resources/testResult/happyPath/discrete_ground_water_old.csv

```diff
@@ -0,0 +1,10 @@
+"8BDA141822744BA5E0530100007FD075","393215104490001","2019-06-20 16:34:00","2019-06-20 16:40:01","EIN, EN, GJ, GJS",,"clear, hot, no precip, very light wind","true","{\"LevelsPerformed\": false, \"CollectionAgency\": \"USGS\", \"OtherSampleTaken\": false, \"SedimentSampleTaken\": false, \"BiologicalSampleTaken\": false, \"RecorderDataCollected\": false, \"WaterQualitySampleTaken\": false, \"GroundWaterLevelPerformed\": true, \"SafetyInspectionPerformed\": false}","2020-05-11 18:35:35.067498","Water level, depth MP","GW level, steel tape","{\"Display\": \"120.28\", \"Numeric\": 120.28}","ft","{\"Display\": \"0.01\", \"Numeric\": 0.01}","ReferencePrimary","Unspecified","Unspecified","HIF-242820","2019-06-20 16:36:00","Source: Reporting Agency","true","true","2a9e559615e04ca7b7191d5ca99e7332","false","Static","[\"Static\"]","{\"Cut\": {}, \"Hold\": {}, \"WaterLevel\": {}, \"TapeCorrection\": {}}"
+"8BDA141822744BA5E0530100007FD075","393215104490001","2019-06-20 16:34:00","2019-06-20 16:40:01","EIN, EN, GJ, GJS",,"clear, hot, no precip, very light wind","true","{\"LevelsPerformed\": false, \"CollectionAgency\": \"USGS\", \"OtherSampleTaken\": false, \"SedimentSampleTaken\": false, \"BiologicalSampleTaken\": false, \"RecorderDataCollected\": false, \"WaterQualitySampleTaken\": false, \"GroundWaterLevelPerformed\": true, \"SafetyInspectionPerformed\": false}","2020-05-11 18:35:35.067498","Water level, depth LSD","GW level, calib steel tape","{\"Display\": \"120.28\", \"Numeric\": 120.28}","ft","{}","ReferencePrimary","Unspecified","Unspecified","Unspecified (GWSI Discrete)","2019-06-20 16:40:00",null,"false","true",null,"false",null,null,"{\"Cut\": {}, \"Hold\": {}, \"WaterLevel\": {}, \"TapeCorrection\": {}}"
+"8BDA141822744BA5E0530100007FD075","393215104490001","2019-06-20 16:34:00","2019-06-20 16:40:01","EIN, EN, GJ, GJS",,"clear, hot, no precip, very light wind","true","{\"LevelsPerformed\": false, \"CollectionAgency\": \"USGS\", \"OtherSampleTaken\": false, \"SedimentSampleTaken\": false, \"BiologicalSampleTaken\": false, \"RecorderDataCollected\": false, \"WaterQualitySampleTaken\": false, \"GroundWaterLevelPerformed\": true, \"SafetyInspectionPerformed\": false}","2020-05-11 18:35:35.067498","Water level, depth LSD","GW level, calib steel tape","{\"Display\": \"120.28\", \"Numeric\": 120.28}","ft","{}","ReferencePrimary","Unspecified","Unspecified","Unspecified (GWSI Discrete)","2019-06-20 16:36:00",null,"false","true",null,"false",null,null,"{\"Cut\": {}, \"Hold\": {}, \"WaterLevel\": {}, \"TapeCorrection\": {}}"
```

Comment:
This file will not be staying.  However, it shows that at least the publish filter is working...